sdk/communication/azure-communication-chat/swagger/README.md

```diff
@@ -33,13 +33,13 @@ To update generated files for chat service, run the following command
 ``` yaml
 tag: package-chat-2021-04-05-preview6
 require:
-    -  https://raw.githubusercontent.com/Azure/azure-rest-api-specs/896d05e37dbb00712726620b8d679cc3c3be09fb/specification/communication/data-plane/Chat/readme.md
+    - https://github.com/Azure/azure-rest-api-specs/blob/2737ef83c687cd61721ece7af713921d0df2485a/specification/communication/data-plane/Chat/preview/2021-04-05-preview6/communicationserviceschat.json
```

Comment:
Why are we switching pattern here? It'll be better if we keep the same pattern. In other words, should still use the "readme.md" file here.

Reply:
Updated link now